src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -607,5 +608,22 @@
 					"\t\tsend \"Oops! Mending failed!\" to player")
 				.since("2.5.1");
 		}
+		Skript.registerEvent("Anvil Prepare", SimpleEvent.class, PrepareAnvilEvent.class, "anvil prepar(e|ing)")
+			.description("Called when an item is put in a slot for repair by an anvil. Please not that this event is called multiple times in a single item slot move.")
+			.requiredPlugins("Minecraft 1.9 or newer")
+			.examples("on anvil prepare:",
+				"\tevent-item is set # result item",
+				"\tchance of 5%:",
+				"\t\tset repair cost to repair cost * 50%",
+				"\t\tsend \"You're are LUCKY! You got 50% discount.\" to player")
+			.since("INSERT VERSION");
+		if (Skript.classExists("com.destroystokyo.paper.event.block.AnvilDamagedEvent")) {
+			Skript.registerEvent("Anvil Damage", SimpleEvent.class, AnvilDamagedEvent.class, "anvil damag(e|ing)")
+				.description("Called when an anvil is damaged/broken from being used to repair/rename items.")
```

Comment:
I checked, this event doesn't cover anvil damage from falling (this is apparently under `EntityChangeBlockEvent`) but I am wondering if it might be worth noting here that it will *not* be run for anvils falling, so people aren't confused why it isn't running.